### PR TITLE
Replace hardcode of server status json

### PR DESCRIPTION
**Pull request by aleksey-hariton**
Add constant for ChefServer name in running status check

## Description

Add constant for ChefServer status file, related to https://github.com/chef/chef-server/pull/3101

### CODE REVIEW

**Review by PrajaktaPurohit — APPROVED** [review]

File: lib/omnibus-ctl.rb [hunk 1]
```diff
@@ -437,8 +437,9 @@ def package_name
 
     # returns nil when chef-server-running.json does not exist
     def running_config
-      @running_config ||= if File.exist?("#{etc_path}/chef-server-running.json")
-                            JSON.parse(File.read("#{etc_path}/chef-server-running.json"))
+      fname = "#{etc_path}/#{::ChefUtils::Dist::Server::SERVER}-running.json"
```
Comment (tas50) [inline lib/omnibus-ctl.rb:440]: This is missing the require of chef utils that makes it work. Check the failing specs

**Review by tas50 — APPROVED** [review]

File: lib/omnibus-ctl.rb (hunk 1 above)
Comment (aleksey-hariton) [reply lib/omnibus-ctl.rb:441]: My fault, fixed @tas50.

**Review by PrajaktaPurohit — APPROVED** [review]

**Comment by aleksey-hariton** [comment]
@tas50 @PrajaktaPurohit who can merge it?

**Comment by aleksey-hariton** [comment]
@scotthain @stevendanna mb you can merge it?

**Comment by ramereth** [comment]
Can we please get this merged/released soon? This is holding back us getting a proper Cinc Server build going Thanks!

**Comment by PrajaktaPurohit** [comment]
I have the magical powers now! My apologies it took a bit to get sorted out.

**Pull request merged**